src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5664,6 +5952,82 @@ namespace ZoneTempPredictorCorrector {
 	}
 
 	void
+	AdjustOperativeSetPointsforAdapComfort(
+		int const TempControlledZoneID,
+		Real64 & ZoneAirSetPoint
+		)
+	{
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Xuan Luo
+		//       DATE WRITTEN   Jan 2017
+		//       MODIFIED       na
+		//       RE-ENGINEERED  na
+
+		// PURPOSE OF THIS SUBROUTINE:
+		// This routine adjust the operative setpoints for each controlled adaptive thermal comfort models.
+
+		// METHODOLOGY EMPLOYED:
+		// na
+
+		// REFERENCES:
+		// na
+
+		// Using/Aliasing
+		using DataEnvironment::DayOfYear;
+		using WeatherManager::DesDayInput;
+		using WeatherManager::Envrn;
+		using WeatherManager::Environment;		
+
+		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+		int originZoneAirSetPoint = ZoneAirSetPoint;
+		int AdaptiveComfortModelTypeIndex = TempControlledZone( TempControlledZoneID ).AdaptiveComfortModelTypeIndex;
+
+		// FLOW:
+		// adjust zone operative setpoint
+		if ( !( TempControlledZone( TempControlledZoneID ).AdaptiveComfortTempControl ) ) return; // do nothing to setpoint
+		if ( ( Environment( Envrn ).KindOfEnvrn != ksDesignDay ) && ( Environment( Envrn ).KindOfEnvrn != ksHVACSizeDesignDay ) ) {
+			// Adjust run period cooling set point 
+			switch ( AdaptiveComfortModelTypeIndex ) {
+				case ASH55_CENTRAL:
+					ZoneAirSetPoint = AdapComfortDailySetPointSchedule.ThermalComfortAdaptiveASH55_Central( DayOfYear );
```

Comment:
That is correct. In run time simulation, we overwrite the operative temperature first, and the dry air temperature based on radiative fraction is calculated afterwards,